contracts/v0.2/src/SubmissionProxy.sol

```diff
@@ -20,15 +20,22 @@ contract SubmissionProxy is Ownable {
     uint256 public constant MAX_SUBMISSION = 1_000;
     uint256 public constant MIN_EXPIRATION = 1 days;
     uint256 public constant MAX_EXPIRATION = 365 days;
+    uint8 public constant MIN_THRESHOLD = 1;
+    uint8 public constant MAX_THRESHOLD = 100;
 
     uint256 public maxSubmission = 50;
     uint256 public expirationPeriod = 5 weeks;
+    uint8 public threshold = 50; // 50 %
+    address[] public oracles;
+
     mapping(address oracle => uint256 expirationTime) public whitelist;
```

Comment:
How about keeping the index mapping from the contract and not taking from submit parameter?

```suggestion
   struct oracleInfo {
   index uint8;
   expirationTime uint256;
   }

    mapping(address oracle => oracleInfo info) public whitelist;
```

Reply:
Updated at https://github.com/Bisonai/orakl/pull/1379/commits/a54e11ddccc1d3e2fec580ba0db18bdc3c0b699f